### ISSUE TITLE
nicehash restart minner almost all the time

### ISSUE THREAD

**Issue by ghost** [nicehash/NiceHashMiner-Archived#1142]
Anything that i can do? Got 6 x 1060 nvidia cards. Miner restart very ofter or all the time

`[pid(4836)|bin(bin\ethminer\ethminer.exe)] READ SPEED ZERO, will cool up
[pid(4836)|bin(bin\ethminer\ethminer.exe)] Cooling UP, cool time is 10000 ms
[pid(4836)|bin(bin\ethminer\ethminer.exe)] Cooling UP, cool time is 20000 ms
NICEHASH] SMA get
[pid(4836)|bin(bin\ethminer\ethminer.exe)] Miner_Exited Will restart in 5000 ms
[pid(4836)|bin(bin\ethminer\ethminer.exe)] Could not read data from API bind port
GetSummary returned null..
[pid(4836)|bin(bin\ethminer\ethminer.exe)] Restarting miner..
[pid(4836)|bin(bin\ethminer\ethminer.exe)] Shutting down miner
Starting miner [pid(3408)|bin(bin\ethminer\ethminer.exe)]  --cuda  -S daggerhashimoto.eu.nicehash.com:3353 -O 1KJkC3VAdZHxQhk5VbCcDtUU8kqV1k2EpX.worker1:x  --api-port 4001 --cuda-devices  0 1 2 3 4 5 --dag-load-mode singlekeep 0`

**Comment by OlorinTharkun** [comment]
Yes!!! Help, help help!!! OK, I'll "cool up" a little.  But seriously, I'm having exactly the same issue, which is making it completely pointless to have CPU mining active.  Until this is fixed, I'm only running GPU mining (3 x 1060 6GB in one computer; 2x same card in another computer).  Both rigs have exact same issue of "read speed zero... cool up... restarting"  Additional discussion here:

[https://github.com/nicehash/NiceHashMiner2/issues/129](url)
